contracts/payment/RefundEscrow.sol

```diff
@@ -0,0 +1,91 @@
+pragma solidity ^0.4.23;
+
+import "./ConditionalEscrow.sol";
+import "../ownership/Ownable.sol";
+
+
+/**
+ * @title RefundEscrow
+ * @dev Escrow that holds investor funds for a unique benefitiary, and allows for
+ * either withdrawal by the benefiatiary, or refunds to the investors.
```

Comment:
Typo twice here: "benefitiary" → "beneficiary".

Also, "unique" doesn't sound right... "single" is the right word, but it feels redundant to me anyway.

This description doesn't explain the condition according to which either withdrawal or refunding is allowed. We should also explain that this contract was written as the escrow for a `RefundableCrowdsale`.

Reply:
Yeah, `unique` is just plain wrong. I think it could just say 'for a beneficiary' and the meaning would remain clear.

I'm not sure about the `RefundableCrowdsale` part though: supporting refunds in a payment system doesn't scream 'crowdsale' to me. I actually see the inverse of that being true: the `RefundableCrowdsale` is nothing more than a crowdsale, the escrow, and glue code.

Reply:
It's true that `RefundEscrow` is not necessarily crowdsale-specific. I didn't mean to say that. I'm just interested in cross-linking related contracts. Perhaps this information should be automatically generated by our documentation generator though.